client/ayon_blender/api/pipeline.py

```diff
@@ -220,12 +221,34 @@ def message_window(title, message):
     _process_app_events()
 
 
-def get_folder_attributes():
-    project_name = get_current_project_name()
-    folder_path = get_current_folder_path()
-    folder_entity = ayon_api.get_folder_by_path(project_name, folder_path)
+def get_frame_range(task_entity=None) -> Union[Dict[str, Any], None]:
+    """Get the current task frame range and handles
+
+    Args:
+        task_entity (dict): Task Entity.
 
-    return folder_entity["attrib"]
+    Returns:
+        dict: with frame start, frame end, handle start, handle end.
```

Comment:
This return type should actually match the return type above.

```suggestion
        Union[Dict[str, Any], None]: Dictionary with 
            frame start, frame end, handle start, handle end.
```

Yet, it also seems that all the values are `int` so it should be:
```suggestion
        Union[Dict[str, int], None]: Dictionary with 
            frame start, frame end, handle start, handle end.
```